uwblueprint/social-diversity-for-children
# Neel/integate zoom meeting links for classes

https://user-images.githubusercontent.com/23081661/137661187-df149360-538f-49c9-a00e-23f673048899.mov

### Notion ticket link

Integrate Zoom Meeting Links For Classes (https://www.notion.so/uwblueprintexecs/Integrate-Zoom-Meeting-Links-For-Classes-70007a22e14c4ca98c15d68d135290aa)

### Implementation description

- created a Parameter Store in AWS to keep track of the Zoom link being used for virtual classes
- created ssm instance and function to retrieve the parameter store from AWS
- created API endpoint which uses the above function to get the Zoom link and pass it to the client
- created custom hook to call the API and retrieve Zoom link
- link retrieved by custom hook is passed as href value for "Join Class" button

### Steps to test

1. Login in to an account
2. Manually make yourself a Volunteer for any class using the Prisma web client
3. Go to the "My Classes" page
4. Click the "Join Class" button and ensure the Zoom landing page is opened in a new tab

### What should reviewers focus on?

- Ensure Zoom meeting opens after clicking "Join Class" button

### Checklist
